src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceModelHttpMessageHandler.Windows.cs

```diff
@@ -39,6 +39,12 @@ public bool UseCookies
             }
         }
 
+        public CookieContainer CookieContainer
+        {
+            get { return _innerHandler.CookieContainer; }
+            set { _innerHandler.CookieContainer = value; }
+        }
```

Comment:
I don't see any difference in the CookieContainer {get;set;} properties between the three handers. Can we move this up to ServiceModelHttpMessageHandler.cs?